rxnorm-integration/src/test/java/dev/ikm/tinkar/rxnorm/integration/RxnormDescriptionSemanticIT.java

```diff
@@ -0,0 +1,190 @@
+        String absolutePath = rxnormOwlFileName;
+        int notFound = processOwlFile(absolutePath, errorFile);
+
+        assertEquals(0, notFound, "Unable to find " + notFound + " Rxnorm Description semantics. Details written to " + errorFile);
+    }
+
+    @Override
+    protected boolean assertOwlElement(RxnormData rxnormData) {
+        if (rxnormData.getId() != null) {
+            // Generate UUID based on RxNorm ID
+            UUID rxnormUuid = conceptUuid(rxnormData.getId());
+            EntityProxy.Concept concept = EntityProxy.Concept.make(PublicIds.of(rxnormUuid));
+            StateSet stateActive = StateSet.ACTIVE;
+            StampCalculator stampCalcActive = StampCalculatorWithCache
+                    .getCalculator(StampCoordinateRecord.make(stateActive, Coordinates.Position.LatestOnDevelopment()));
+            PatternEntityVersion latestDescriptionPattern = (PatternEntityVersion) Calculators.Stamp.DevelopmentLatest().latest(TinkarTerm.DESCRIPTION_PATTERN).get();
+            AtomicBoolean rxnormName = new AtomicBoolean(!rxnormData.getRxnormName().isEmpty());
+            AtomicBoolean rxnormSynonym = new AtomicBoolean(!rxnormData.getRxnormSynonym().isEmpty());
+            AtomicBoolean rxnormPrescribableSynonym = new AtomicBoolean(!rxnormData.getPrescribableSynonym().isEmpty());
+            AtomicReference<List<String>> rxnormTallmanSynonym = new AtomicReference<>();
+
+            if (!rxnormData.getTallmanSynonyms().isEmpty()) {
+                rxnormTallmanSynonym.set(rxnormData.getTallmanSynonyms());
+            } else {
+                rxnormTallmanSynonym.set(new ArrayList<>());
+            }
+
+            AtomicBoolean matchedName = new AtomicBoolean(!rxnormName.get());
+            AtomicBoolean matchedSynonym = new AtomicBoolean(!rxnormSynonym.get());
+            AtomicBoolean matchedPrescribableSynonym = new AtomicBoolean(!rxnormPrescribableSynonym.get());
+            AtomicInteger matchedTallmanSynonyms = new AtomicInteger(0);
+
+            AtomicInteger counter = new AtomicInteger(0);
+
+            EntityService.get().forEachSemanticForComponentOfPattern(concept.nid(), TinkarTerm.DESCRIPTION_PATTERN.nid(), semanticEntity -> {
+                Latest<SemanticEntityVersion> latestActive = stampCalcActive.latest(semanticEntity);
+
+                counter.incrementAndGet();
+                if (latestActive.isPresent()) {
+                    String textForDesc = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.TEXT_FOR_DESCRIPTION, latestActive.get());
+                    Component descCaseSignificance = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_CASE_SIGNIFICANCE, latestActive.get());
+                    Component descType = latestDescriptionPattern.getFieldWithMeaning(TinkarTerm.DESCRIPTION_TYPE, latestActive.get());
+
+                    if (rxnormName.get()) {
+                        if (rxnormData.getId().equals("1000001")) {
```

Comment:
You can remove all of the explicit references to "1000001" and anything with "JTD" in the output. You don't need the info statements.